packagingbuild/Dockerfile.debian

```
@@ -16,7 +16,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
 
 # Install fresh pip and co
 {% if version in ('bionic') -%}
-RUN curl https://bootstrap.pypa.io/get-pip.py | python3 - virtualenv==16.6.0 pip==19.1.1 wheel setuptools; \
+RUN curl https://bootstrap.pypa.io/get-pip.py | python3 - virtualenv==16.6.0 pip==19.1.1 wheel setuptools cryptography==3.2; \
```

Do we need to pin it to 3.2 or do we just need to specify it? (Serious question)

```suggestion
RUN curl https://bootstrap.pypa.io/get-pip.py | python3 - virtualenv==16.6.0 pip==19.1.1 wheel setuptools cryptography; \
```

---

Looks like just specifying cryptography up front resolves the issues, we end up initially then with crypto 3.3 and then it seems to manage to build the RPM correctly (and pulls in the requirements from ST2 of crypto==3.2). Have pushed up a new commit.